src/main/java/com/carina/methods/mobile/gui/demoblaze/components/AboutUsVideo.java

```diff
@@ -0,0 +1,38 @@
+package com.carina.methods.mobile.gui.demoblaze.components;
+
+import com.carina.methods.api.PutEmployeeMethod;
+import com.carina.methods.mobile.gui.demoblaze.utils.Utils;
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.gui.AbstractUIObject;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+
+import java.security.PublicKey;
+
+
+public class AboutUsVideo extends AbstractUIObject {
+
+
+    @FindBy(xpath = "//div[@id='example-video']")
+    private ExtendedWebElement playButton;
+
+    @FindBy(xpath = "//div [@class = 'video-js example-video-dimensions vjs-controls-enabled vjs-touch-enabled vjs-workinghover vjs-v7 vjs-has-started vjs-paused vjs-user-inactive']")
+    private ExtendedWebElement videoPaused;
+
+    public AboutUsVideo(WebDriver driver) {
+        super(driver);
+    }
+
+    public void clickOnPlayButton(){
+        Utils.waitForElementVisible(getDriver(), playButton);
+        playButton.click();
+    }
+
+
+    public boolean playButtonInvisible(){
+        return !videoPaused.isVisible();
+    }
+
```

Comment:
1. If your method returns boolean in 90% cases it should have name isSomethingPresent or isSomethingActive etc
2. Try to avoid usage of `!` here because it may confuse you and your teammates when you support tests. If you need to check that it's not present just just use assertFalse in the test
```suggestion
    public boolean isPlayButtonPresent() {
        return isElementPresent();
    }
```